lib/components/input-group.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div :class="classObject">
+        <slot name="left"></slot>
```

Comment:
You are true, we can provide both methods and let users choose best one that fits to their usages :)